Makefile

```diff
@@ -32,5 +37,23 @@ bootstrap-terraform: ## Bootstrap local environment for first use
 	@{ \
 		export AWS_PROFILE=$(aws_profile); \
 		export AWS_REGION=$(aws_region); \
-		python bootstrap_terraform.py; \
+		python3 bootstrap_terraform.py; \
 	}
+
+.PHONY: bootstrap-terraform-dev
+bootstrap-terraform-dev: ## Bootstrap local environment for first use
+	make bootstrap-terraform
+	/usr/bin/sed -i '' 's|"default" ? "management-dev"|"default" ? "development"|g' terraform/deploy/terraform.tf
+	/usr/bin/sed -i '' 's|terraform/dataworks/aws-concourse.tfstate|terraform/dataworks/aws-concourse-${shell git rev-parse --abbrev-ref HEAD}.tfstate|g' terraform/deploy/terraform.tf
```

Comment:
We should just use `sed` here? it might not be in `user/bin` on all users' PCs

Reply:
this needs to be done differently now that our admin access has changed